### ISSUE TITLE
The little-endian and big-endian of scriptHash is not consistent

### ISSUE THREAD

**Issue by chenquanyu** [neow3j/neow3j#75]
The ContractInvocation uses big-endian contractScriptHash as parameter, at the same time the address of a smart contrtact should be encoded with little-endian scriptHash

```
    public static String scriptHashToAddress(String input) {
        byte[] inputBytes = Numeric.hexStringToByteArray(input);
        return toAddress(inputBytes);
    }
```

**Comment by chenquanyu** [comment]
The hex string in stack value with ByteArray is also little-endian
![image](https://user-images.githubusercontent.com/4393234/61930373-1a307900-afb0-11e9-9cde-a06644604c59.png)

**Comment by gsmachado** [comment]
This is published on the `2.1.0-SNAPSHOT`. It will be included on the `2.1.0` release.
